src/openapi/AccountManager.yaml

```diff
@@ -93,7 +93,7 @@ paths:
         description: Initial password for the email account. The password is base64 encoded.
         required: true
         schema:
-          type: string
+          type: byte
```

Comment:
Hier auch nochmal.

type: string
format: byte

Reply:
Ja das ist richtig, aber wäre hier 'password' nicht doch besser?
String Formats ->
password – a hint to UIs to mask the input
byte – base64-encoded characters, for example, U3dhZ2dlciByb2Nrcw==